adaptors/ampersand/base_view.js

```diff
@@ -31,10 +31,6 @@ var BaseView = AmpersandView.extend({
     }
     this.options = options || {};
 
-    // VTree is passable as an option if we are transitioning in from a different view
```

Comment:
Was this intentional ? Seems like you can't pass in an initial vtree when using Ampersand ?